common/management/commands/create_test_data.py

```diff
@@ -580,7 +581,7 @@ def _create_service_sector_roles(
     text_entering="Creating units...",
     text_exiting="Units created!",
 )
-def _create_units(*, number: int = 15) -> list[Unit]:
+def _create_units(*, number: int = 300) -> list[Unit]:
```

Comment:
Was this supposed to be `30`? `300` units is a lot 😄

Reply:
Previously, we generated 15 units and 300 reservation units, and used a unit randomly for each of them. Now we generate one unit per reservation unit, which for frontend testing was deemed easier.